### ISSUE TITLE
Rule Proposal: `space-after-function-paren`

### ISSUE THREAD

**Issue by robertkowalski** [eslint/eslint#4163]
It would be nice if we could also test for spaces after the function parentheses.

rule-type: stylistic.

why in core? it is complimentary to the existing rule which tests for spaces before the function parentheses

Are you willing to create the rule yourself?

Yes, I just have to get an OK from my manager and I hope I get it. It is probably easier and faster if someone else sends the PR.

E.g:

```js
function foo () {

}

// will warn if space-after-function-paren: 2, never
```

vs.:

```js
function foo (){

}

// will warn if space-after-function-paren: 2, always
```

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by alberto** [comment]
I think space-before-blocks is what you are looking for.

**Comment by nzakas** [comment]
Yup, see: http://eslint.org/docs/rules/space-before-blocks
